geode-core/src/integrationTest/java/org/apache/geode/internal/cache/tier/sockets/AcceptorImplIntegrationTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.cache.tier.sockets;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+import java.io.IOException;
+import java.net.BindException;
+import java.util.Collections;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import org.apache.geode.cache.CacheException;
+import org.apache.geode.cache.CacheFactory;
+import org.apache.geode.cache.server.CacheServer;
+import org.apache.geode.internal.cache.InternalCache;
+import org.apache.geode.test.junit.categories.ClientServerTest;
+
+@Category(ClientServerTest.class)
+public class AcceptorImplIntegrationTest {
+
+  private InternalCache cache;
+  private AcceptorImpl acceptor1;
+  private AcceptorImpl acceptor2;
+  private ServerConnectionFactory serverConnectionFactory;
+
+  @Before
+  public void setUp() throws Exception {
+    cache = (InternalCache) new CacheFactory().create();
+    serverConnectionFactory = new ServerConnectionFactory();
+  }
+
+  @After
+  public void tearDown() throws Exception {
+    if (acceptor1 != null) {
+      acceptor1.close();
+    }
+    if (acceptor2 != null) {
+      acceptor2.close();
+    }
+    cache.close();
+  }
+
+  @Test
+  public void constructorThrowsBindException() throws CacheException, IOException {
```

Comment:
Under what conditions does the constructor throw? Add the condition(s) to the test name.

Reply:
I renamed the test to constructorThrowsBindExceptionIfPortIsInUse.